docs/source/version-5-cli.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Configuration with CLI tools in Version 5
+permalink: /v5/configuration-cli-tools/
+toc: true
+---
+
+Coral’s CLI tooling has been reinvented in version 5. We’ve replaced the former `./bin/cli` tools with a completely new CLI application which is available on GitHub at: https://github.com/coralproject/coral-cli
+
+The new CLI was built on `oclif` ([View on GitHub](https://github.com/oclif)), the open source CLI Framework originally built by Heroku, and offers an extensible platform for interacting with Coral via CLI. 
+
```

Comment:
I would not suggest copying the README of the cli tool, and instead direct anyone to the https://github.com/coralproject/coral-cli repo because it's README is automatically generated by the code.

Reply:
I didn't actually copy the readme, except to list the commands, and I did add additional descriptions to provide greater context than what is available on the readme. 

I agree self generating docs are the way to go everywhere we can, but if you compare this to the readme it's not 1:1. While the readme and the built in help outputs are great, I think they need supplementing with things I've added here, like for example when you'd use a command, and more details of what the results of use will be.